tests/compose-rootfs/Containerfile

```diff
@@ -1,5 +1,5 @@
-# In theory this can skew from the builder
-FROM quay.io/fedora/fedora:41 as repos
+# Demonstrate skew from the builder
+FROM quay.io/centos/centos:stream10 as repos
 
 # You must run this build with `-v /path/to/rpm-ostree:/run/build/rpm-ostree:ro`
```

Comment:
Hmm, how about instead we mount an install tree and then `rsync` that in the container env? Similar to https://github.com/coreos/rpm-ostree/pull/5286.

Reply:
Yeah the CI sprawl is a huge pain. Lately what I've been thinking a lot about is getting really strict and saying: all build stuff is always in a container.

OK to just leave this bit for now and circle back to the larger CI problem...well, not sure when :cry: